internal/controller/controllers/clusterdeployments_controller.go

```diff
@@ -877,6 +913,12 @@ func (r *ClusterDeploymentsReconciler) createNewCluster(
 		clusterParams.ServiceNetworkCidr = swag.String(clusterInstall.Spec.Networking.ServiceNetwork[0])
 	}
 
+	if network.IsIPv6CIDR(swag.StringValue(clusterParams.ClusterNetworkCidr)) || network.IsIPv6CIDR(swag.StringValue(clusterParams.ServiceNetworkCidr)) {
```

Comment:
if it's not required then maybe we should have this logic only update if needed?

Reply:
`NetworkType` is not required in creation so we can remove this part.
Update will be called immediately after.